moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -559,11 +559,17 @@ bool ServoCalcs::cartesianServoCalcs(geometry_msgs::msg::TwistStamped& cmd,
     delta_theta_ = pseudo_inverse * delta_x;
   }
 
+  auto last_status = status_;
   delta_theta_ *= velocityScalingFactorForSingularity(joint_model_group_, delta_x, svd, pseudo_inverse,
                                                       servo_params_.hard_stop_singularity_threshold,
                                                       servo_params_.lower_singularity_threshold,
                                                       servo_params_.leaving_singularity_threshold_multiplier,
-                                                      *node_->get_clock(), current_state_, status_);
+                                                      current_state_, status_);
+  // Status will have changed if approcaching singularity
```

Comment:
```suggestion
  // Status will have changed if approaching singularity
```

Also it is worth actually printing something in front of it like "Servo status changed to BLAH"

Reply:
Not sure about that since it already prints a verbose message for the statuses when near singularity, would that not be enough ?

```c++
{ StatusCode::DECELERATE_FOR_LEAVING_SINGULARITY, "Moving away from a singularity, decelerating" },
{ StatusCode::DECELERATE_FOR_APPROACHING_SINGULARITY, "Moving closer to a singularity, decelerating" },
{ StatusCode::HALT_FOR_SINGULARITY, "Very close to a singularity, emergency stop" },
```

Reply:
Ah yeah I think that's fine then. Thanks!